ui/src/components/admin/DataDictionary/DataDictionary.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { AdminContainer, AdminHeader, AdminHeaderH1, AdminHeaderBlock } from 'theme/adminStyles';
+import { NotificationToaster } from '../Notifications';
+
+import TabGroup from 'components/layout/VerticalTabs';
+
+const DataDictionary = () => {
+  // temporarily hardcoding some dummy tabs
+  const groupName = 'Example Vertical Tab Group';
+  const tabs = [
+    {
+      id: 'tab-0',
+      heading: 'Active',
+      subheading: 'Can set a tab to be active initially',
+    },
+    {
+      id: 'tab-1',
+      heading: 'Default',
+      subheading: 'With subheading',
+    },
+    {
+      id: 'tab-2',
+      heading: 'Has Dot',
+      subheading: 'With status indicator | More text',
+      dot: true,
+    },
+    {
+      id: 'tab-3',
+      heading: 'No Subheading',
+    },
+    {
+      id: 'tab-4',
+      heading: 'Disabled',
+      subheading: 'No click, no hover, no problem',
+      disabled: true,
+    },
+  ];
+
+  return (
+    <AdminContainer>
+      <NotificationToaster />
+      <AdminHeader>
+        <AdminHeaderH1>Data Dictionary</AdminHeaderH1>
+        <AdminHeaderBlock />
+      </AdminHeader>
+      <TabGroup groupName={groupName} tabs={tabs} initialActive={tabs[0].id} />
```

Comment:
Seeing this in practice, I'm wondering if having it built with <TabItem /> components inside the TabGroup would be better to maintain consistency:
```
<TabGroup groupName={groupName}  initialActive={tabs[0].id} >
  <TabItem id="tab-1" heading="Active" subheading="Can set a tab..." />
  <TabItem id="tab-2" heading="Disabled" disabled={true} />
  <TabItem id="tab-3" heading="Has Dot" dot={true} />
</TabGroup>
```

Additional benefit is that it doesn't have to re-render the entire tab group when a dot or disabled status of one of the items changes.

Reply:
That can be done for sure. Would definitely keep it more consistent with things like `ui/src/components/admin/DataDictionary/index.js` (https://github.com/nci-hcmi-catalog/portal/pull/575/files#diff-2e15a0b5addf568f34c6b21af10aec72)

I just wasn't sure if we'd be getting the fields programmatically or not